src/test/java/com/artipie/rpm/TestRpm.java

```diff
@@ -203,6 +212,42 @@ public Path path() {
 
     }
 
+    /**
+     * Get file by name from test resources.
+     * @since 0.11
+     */
+    final class TestResource {
```

Comment:
@olenagerasimova I would extract this class to a separate file, it would be easier to use and would me more visible to other developers since it is not really a part of `TestRpm`.